client/src/featureform/register.py

```diff
@@ -1316,6 +1322,26 @@ def register_column_resources(
             label_resources.append(resource)
         return ResourceRegistrar(self, features, labels)
 
+    def __get_feature_nv(self, features):
+        feature_nv_list = []
+        for feature in features:
+            if isinstance(feature, dict):
+                feature_nv = (feature["name"], feature["variant"])
+                feature_nv_list.append(feature_nv)
+            elif isinstance(feature, list):
+                feature_nv_list.extend(self.__get_feature_nv(feature))
+            else:
+                feature_nv_list.append(feature)
+        return feature_nv_list
+    
+    def __get_label_nv(self, label):
+        if isinstance(label, list):
+            if len(label) != 1:
```

Comment:
We want this logic inside the label() function, otherwise it won't allow us to register more than one label from a resource.